etc/kayobe/kolla-image-tags.yml

```diff
@@ -4,25 +4,5 @@
 # where the key is the OS distro and the value is the tag to deploy.
 kolla_image_tags:
   openstack:
-    rocky-9: 2023.1-rocky-9-20240621T104542
+    rocky-9: 2023.1-rocky-9-20240709T132012
     ubuntu-jammy: 2023.1-ubuntu-jammy-20240621T104542
-  bifrost_deploy:
-    rocky-9: 2023.1-rocky-9-20240423T125905
-    ubuntu-jammy: 2023.1-ubuntu-jammy-20240423T125905
```

Comment:
Are you sure this should be deleted?

Reply:
ubuntu-jammy should not . 
Thanks for pointing that out.